src/App.js

```diff
@@ -1,25 +1,77 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component } from 'react'
+import Container from './Components/Container'
+import Header from './Components/Header'
+import "./Styles/main.css"
+
+class App extends Component {
+
+  constructor(props) {
+    super(props)
+  
+    this.state = {
+        tasks: [
+            {
+              id: 1,
+              text: "Meeting with Randy",
+              date: "2021-08-09"
+            },
+            {
+              id: 2,
+              text: "Meeting with Randy",
+              date: "2021-08-09"
+            },
+            {
+              id: 3,
+              text: "Meeting with Randy",
+              date: "2021-08-09"
+            }
+          ],
+        isUpdateMode: false,
+        taskToUpdate: {}
+    }
+
+    this.addTasks = this.addTasks.bind(this)
+    this.deleteTask = this.deleteTask.bind(this)
+    this.updateTask = this.updateTask.bind(this)
+    this.setUpdateMode = this.setUpdateMode.bind(this)
```

Comment:
We can remove this binding if we use array functions instead of regular functions, It will clean our constructor function.